devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1658,7 +1661,11 @@ bool WholeBodyDynamicsDevice::attachAll(const PolyDriverList& p)
     ok = ok && this->attachAllControlBoard(p);
     ok = ok && this->attachAllVirtualAnalogSensor(p);
     ok = ok && this->attachAllFTs(p);
-    ok = ok && this->attachAllIMUs(p);
+    
+    if (settings.kinematicSource == IMU)
```

Comment:
If I remember correctly the settings can also be changed via RPC. It probably could be worth to check the other places in which `settings.kinematicSource` is checked, and raise an error if kinematicsSource is switched to IMU after a start without IMU is done.

Reply:
You remember correctly. I agree. I will handle that as well.

Reply:
Fixed in https://github.com/robotology/whole-body-estimators/pull/95/commits/f6c3a8122a6a3aaa4582f2faf8c5ec13bf54c5a2